src/main/java/rs/teslaris/core/service/impl/person/PersonServiceImpl.java

```diff
@@ -582,4 +578,56 @@ private Query buildNameAndEmploymentQuery(List<String> tokens) {
             ))
         )._toQuery();
     }
+
+    private void setAllPersonIdentifiers(Person person, PersonIdentifierable personDTO) {
+        IdentifierUtil.validateAndSetIdentifier(
+            personDTO.getApvnt(),
+            person.getId(),
+            "^\\d+$",
+            personRepository::existsByApvnt,
+            person::setApvnt,
+            "apvntFormatError",
+            "apvntExistsError"
+        );
+
+        IdentifierUtil.validateAndSetIdentifier(
+            personDTO.getECrisId(),
+            person.getId(),
+            "^\\d+$",
+            personRepository::existsByeCrisId,
+            person::setECrisId,
+            "eCrisIdFormatError",
+            "eCrisIdExistsError"
+        );
+
+        IdentifierUtil.validateAndSetIdentifier(
+            personDTO.getENaukaId(),
+            person.getId(),
+            "^[a-zA-Z]{2}\\d+$",
```

Comment:
Consider to change to 
```suggestion
            "^[A-Z]{2}\\d+$",
```